BookingsApi/BookingsApi.Infrastructure.Services/Dtos/ParticipantDto.cs

```diff
@@ -17,5 +19,6 @@ public class ParticipantDto
         public string UserRole { get; set; }
         public CaseRoleGroup CaseGroupType { get; set; }
         public string Representee { get; set; }
+        public IList<LinkedParticipant> LinkedParticipants { get; set; }
```

Comment:
Please follow the models here and not reference the domain LinkedParticipant class